test/test_helper.rb

```diff
@@ -1,5 +1,43 @@
 require 'barnes'
 require 'minitest/autorun'
+require 'pathname'
 
+require 'wait_for_it'
 require 'simplecov'
 SimpleCov.start
+
+def fixture_path(path = "")
+  return Pathname.new(__dir__).join("fixtures").join(path).expand_path
+end
+
+
+require 'socket'
+require 'timeout'
+
+def is_port_open?(ip, port)
+  begin
+    Timeout::timeout(1) do
+      begin
+        s = TCPServer.new(ip, port)
+        s.listen(1024)
+        s.close
+        return true
+      rescue Errno::ECONNREFUSED, Errno::EHOSTUNREACH => e
+        puts e
```

Comment:
This is just a test helper, but did you intend to output the exception?

Reply:
Good catch